Mathlib/GroupTheory/Subgroup/Basic.lean

```diff
@@ -1500,6 +1500,11 @@ theorem map_equiv_eq_comap_symm (f : G ≃* N) (K : Subgroup G) :
 #align subgroup.map_equiv_eq_comap_symm Subgroup.map_equiv_eq_comap_symm
 #align add_subgroup.map_equiv_eq_comap_symm AddSubgroup.map_equiv_eq_comap_symm
 
+@[to_additive]
+theorem map_equiv_eq_comap_symm' (f : G ≃* N) (K : Subgroup G) :
```

Comment:
In my opinion this should be the umprime version, and we can even drop the other one (I didn't check all the version, but it is coherent with the version for ring equiv).

Reply:
Yeah, I agree with that based off of what you said (I haven't researched whether this is the general convention) but curious if others agree?  @eric-wieser @alreadydone ?

Reply:
Will this mess up the #align for mathport (I am completely ignorant of this and whether we can change the names that are currently there -- this would be needed afaict).

Reply:
You should adjust the `#align` to point to the best match for the mathlib3 statement, if one exists. So here, either change the align to point to the primed name (if you keep both), or just leave it alone (if you change the existing existing statement as Riccardo suggests)